static/js/composebox_typeahead.js

```diff
@@ -1123,11 +1124,69 @@ export function initialize() {
         },
         sorter(items) {
             const sorted = typeahead_helper.sorter(this.query, items, (x) => x);
-            if (sorted.length > 0 && !sorted.includes(this.query)) {
+            if (sorted.length >= 0 && !sorted.includes(this.query)) {
                 sorted.unshift(this.query);
             }
             return sorted;
         },
+        process(items) {
+            // item_list contains all the existing topics in a stream. This
```

Comment:
Let's add a comment that we are overriding default `process` and `render` methods and for what what reason.